### ISSUE TITLE
Would you like to provide weights for real defocus scene?

### ISSUE THREAD

**Issue by benhenryL** [limacv/Deblur-NeRF#28]
Hi teams, thank you for sharing your great work!

I want to render some real defocus scenes but unfortunately I cannot train the model with provided batch size because of small gpu memory budget.

So I simply reduced batch size to 512 but the performance degraded (both qualitatively and quantitatively)

So would you like to provide the weight for whole real defocus scenes please? Currently, I can find the only scene 'bush' weight among what you uploaded. Or would you like to let me know which hyperparameters, other than batch size (N_rand), to control to get good result with smaller batch size? 

Thanks!

**Comment by limacv** [comment]
Hi, I'm sorry but I'm afraid these additional pretrained models are not reachable now since it's an intern project and I've leaved that company. You could try to reduce the learning rate and increase the training epoch when using a smaller batch size.
